crates/modelardb_common/src/metadata/mod.rs

```diff
@@ -119,22 +141,38 @@ impl MetadataDeltaLake {
         access_key_id: String,
         secret_access_key: String,
     ) -> Result<Self, DeltaTableError> {
-        let location = format!("s3://{bucket_name}/{METADATA_FOLDER}");
+        let location = format!("s3://{bucket_name}");
 
         // TODO: Determine if it is safe to use AWS_S3_ALLOW_UNSAFE_RENAME.
         let storage_options = HashMap::from([
```

Comment:
I assume you are referring to the changed keys in the storage options. It was changed to avoid problems when the keys are passed to `builder.with_config()` since the builders seem to take lower caps configs instead of the upper caps configs that are expected in environment variables. There might be some `to_lower()` call hidden somewhere in the object_store code but it seemed simpler to just change the keys on our side rather than rely on their code. The `aws_` prefix was added for consistency with `aws_s3_allow_unsafe_rename`.